app/src/main/java/net/skyscanner/backpack/demo/compose/Text.kt

```diff
@@ -21,39 +21,118 @@ package net.skyscanner.backpack.demo.compose
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.items
-import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.unit.dp
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import net.skyscanner.backpack.compose.text.BpkText
 import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkDimension
+import net.skyscanner.backpack.demo.BackpackDemoTheme
 
+@Preview
```

Comment:
same for the demo app composables: create a preview for everything for quick iteration. only overhead is having to supply the theme for everything as we've set it up to crash when it isn't wrapped in a theme when accessing typography, colours etc